test/e2e/cypress/support/commands.js

```diff
@@ -23,3 +23,23 @@
 //
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+
+const initializeOpenSidebar = () => cy.setCookie('collapsedSidebar', 'false')
+
+const selectPagination = (itemsPerPage) => {
+    const pagination = [10, 25, 50, 100]
+    cy.get('.pagination-actions button.dropdown-toggle').click()
+    cy.get(`.pagination-actions .dropdown-menu .dropdown-item:nth-child(${pagination.indexOf(itemsPerPage) + 1})`).click()
```

Comment:
As @fabriziosestito told me since "this test is flaky: minute 1" this is due to debounce timer on the pagination plugin. 

If we have instead:
```
const selectPagination = (itemsPerPage) => {
    const pagination = [10, 25, 50, 100]
    cy.get('.pagination-actions button.dropdown-toggle').click()
    cy.wait(1000)   // Wait 1 second between click to avoid debounce
    cy.get(`.pagination-actions .dropdown-menu .dropdown-item:nth-child(${pagination.indexOf(itemsPerPage) + 1})`).click()
}
```

All issues go away. I tried finding a better solution but my limited cypress knowledge didn't help much. Also, we are not alone on this one:

https://stackoverflow.com/questions/61193886/how-do-i-handle-debounced-using-rxjs-debouncetime-event-emitter-with-cypress

Reply:
Yep, I added `cy.wait(100)`. Hope 100ms is enough :sweat_smile: 
We'll increase if needed.
Agreed?

Reply:
we can look at the code of the plugin to find the right spot

Reply:
Even with 500ms I was still getting it. Starting from 900ms I wouldn't get it anymore regardless of how many times I tried but my OCD enjoys more round cyphers 8)